eslint.config.mjs

```diff
@@ -0,0 +1,54 @@
+import globals from "globals";
+import js from '@eslint/js';
+import tsParser from '@typescript-eslint/parser';
+import tsPlugin from '@typescript-eslint/eslint-plugin';
+import reactPlugin from 'eslint-plugin-react';
+import reactHooksPlugin from 'eslint-plugin-react-hooks';
```

Comment:
Does this support eslint v9? https://github.com/facebook/react/issues/28313#issue-2132424076

Reply:
<img width="1118" alt="image" src="https://github.com/user-attachments/assets/b83f6229-399b-4e46-bf82-c348ae9ce0ec">

We could hold off on this until all dependencies support v9 - not a huge deal. We could potentially upgrade prettier though

Reply:
I think we can probably park this until a stable release is out